nomad/worker.go

```diff
@@ -114,18 +114,23 @@ func (w *Worker) run() {
 
 		// Check for a shutdown
 		if w.srv.IsShutdown() {
+			w.logger.Error(fmt.Sprintf("nacking eval ID %v because the server shutdown", eval.ID))
```

Comment:
Use new formatting style
`w.logger.Error(snacking eval because the server is shutting down", "eval", eval)`

Same for all